app/helpers/rebate_forms_helper.rb

```diff
@@ -47,4 +47,8 @@ def rebate_form_boolean_field_to_english(answer)
       answer.to_s.capitalize
     end
   end
+
+  def rebate_form_signature_required?(rebate_form)
+    rebate_form.completed
```

Comment:
the signature is required if the form is completed? 
isn't it the opposite way around?

Reply:
@Br3nda  Ok, that wasn't clear in the code. What is the status of the rebate_form if the sgnature is there and it is ready to be processed?

Reply:
What does "required" mean? To me, it means you don't have it yet. Does it mean something else?

If the form is complete, doesn't that mean you've got the signature already? 

Why would you want `signature_required?` to be true when `rebate_form.completed` is true?
If the rebate_form is completed, that means you've got the signatures. It makes it's no longer required.

Reply:
maybe it should be named `signatures_present?`

Reply:
```suggestion
    rebate_form.completed != true
```

Reply:
I changed it to be called `rebate_form_completed?`